triton-client/src/components/Landing/index.tsx

```diff
@@ -49,9 +49,62 @@ function LandingPage({ isLoggedIn, userName }: UserInformationProps) {
                     />
                 </Tooltip>
             </Content>
-            <Text className="Common-subtitle Landing-user" type="secondary">
-                {isLoggedIn ? `Logged in as ${userName}` : "Logging In.."}
-            </Text>
+            <Content className="landing-page-content">
+                <Text className="Common-subtitle Landing-user" type="secondary">
+                    {isLoggedIn ? `Logged in as ${userName}` : "Logging In.."}
+                </Text>
+                <Typography.Title level={2}>Guide</Typography.Title>
+                <Content className="faq-container">
+                    <div className="faq-section-container">
+                        <Typography.Title level={5}>
+                            1. The Triton app landing page is where you will
+                            always start.
+                        </Typography.Title>
+                        <Typography.Title level={5}>
+                            2. The App logo will help you return to this page
+                            when you click on it.
+                        </Typography.Title>
+                        <Typography.Title level={5}>
+                            3. These are your projects, Close this dialog and
+                            click on a project from this list to continue.
+                        </Typography.Title>
+                    </div>
+                    <div className="faq-section-container">
+                        <Typography.Title level={5}>
+                            4. The second page will have several status of
+                            available space for Globus and SFTP storage
+                            allocated to your project.
+                        </Typography.Title>
+                        <Typography.Title level={5}>
+                            5. You will be able to reset SFTP or Globus password
+                            in the next page. It will be on the top right corner
+                            of the next page.
+                        </Typography.Title>
+                        <Typography.Title level={5}>
+                            6. You will see the datasets that have been released
+                            in Freezeman, relating to the project, will be
+                            displayed in a collapsable container of the next
+                            page.
```

Comment:
😱
You're not supposed to mention Freezeman to the client.

Reply:
"The datasets that have been released by the PMs"
(maybe not PMs, not sure)

Reply:
for download?